.gitignore

```diff
@@ -0,0 +1 @@
+.env
```

Comment:
Don't forget that newline at the end of the file. You may want to look into configuring your editor to do this automatically.